wpilibc/src/main/native/cpp/PowerDistribution.cpp

```diff
@@ -172,6 +172,118 @@ PowerDistribution::Faults PowerDistribution::GetFaults() const {
   return faults;
 }
 
+bool PowerDistribution::Faults::GetBreakerFault(int channel) const {
+  switch (channel) {
+    case 0:
+      return this->Channel0BreakerFault;
```

Comment:
For this and the `StickFaults` version, why is only the channel 0 fault implicitly converted to bool instead of explicitly compared to 0? This isn't that important (They should function identically), just wondering if we want to bother fixing it.

Reply:
Good point. Better to be consistent. I think the implicit conversion is clear enough.